rules/windows/credential_access_domain_backup_dpapi_private_keys.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/13"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies creation or modification of Domain Backup private keys, which may indicate activity related to Domain Backdup DPAPI
+decryption keys access.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Creation or Modification of Domain Backup DPAPI private key"
+note = "### Domain DPAPI Backup keys are stored on domain controllers and can be dumped remotely with tools such as Mimikatz. The resulting .pvk private key can be used to decrypt ANY domain user masterkeys, which then can be used to decrypt any secrets protected by those keys."
+references = ["https://www.dsinternals.com/en/retrieving-dpapi-backup-keys-from-active-directory/"]
```

Comment:
Useful reference as well - https://threathunterplaybook.com/notebooks/windows/06_credential_access/WIN-190620024610.html

Reply:
ref in there is older enough 2015 :)